Rakefile

```diff
@@ -5,11 +5,13 @@ require File.expand_path('../lib/bootstrap-datepicker-rails/version', __FILE__)
 
 desc "Update assets"
 task :update do
+  branch = ARGV.count == 1 ? "tags/#{ARGV.first}" : "master"
+
   if Dir.exist?('bootstrap-datepicker-src')
-    system("cd bootstrap-datepicker-src && git checkout master && git pull && git checkout `git describe --abbrev=0` && cd ..")
+    system("cd bootstrap-datepicker-src && git checkout #{branch} && git pull && git checkout `git describe --abbrev=0` && cd ..")
```

Comment:
@bencolon

The line 'git checkout `git describe --abbrev=0`' will checkout the latest tag on master, which at the moment is v1.4.0. 

I put the `git checkout master` there so that the `git pull` will retrieve from master, as otherwise it may be trying to `git pull` from a detached state – that's the resulting state when you checkout a tag. 

So for this change to be correct, you will need to remove 'git checkout `git describe --abbrev=0`', or otherwise it will always checkout the latest release regardless of the input.

I like this change as it makes it more flexible.